composer.json

```diff
@@ -8,7 +8,7 @@
         "php"                       : ">=7.2.5",
         "guzzlehttp/guzzle"         : "^6.5.6||^7.4.3",
         "guzzlehttp/oauth-subscriber": "~0.3",
-        "symfony/validator"         : "4.*||5.*"
+        "symfony/validator"         : ">=4.0.0 <=7.0.0"
```

Comment:
This constraint will not allow to use `7.0.x` versions, 
My suggestion would be to use this constraint: `"4.0 - 7.0"`, it would also include 7.0.x versions.
https://semver.madewithlove.com/?package=symfony%2Fvalidator&constraint=4.0+-+7.0

Reply:
updated